cmd/result.go

```diff
@@ -59,6 +59,11 @@ func createFields(res Result, occ Occurrence) (fields log.Fields) {
 		fields[k] = v
 	}
 	fields["Container"] = occ.container
+
```

Comment:
Now that we have errors that are not related to a container we should make sure to only set the container field if the container is actually present.

```
ERRO[0000] RunAsNonRoot is set to false (root user allowed), please set to true!  Container= KubeType=pod Name=run_as_non_root_false Namespace=fakeDeploymentRANR Pod=fakeRANR
```

Reply:
this has been fixed. Great Suggestion!